packages/hummingbird/src/components/Navigation/ChaptersToggle.js

```diff
@@ -62,11 +62,11 @@ export default class ChaptersToggle extends React.Component {
   };
 
   render() {
-    const { locale } = this.props;
+    const { title } = this.props;
```

Comment:
There is currently no title prop being passed in to the component:

https://github.com/wingscms/wings/blob/6f9cb706232fb5ae183ca88c7401969e6767720c/packages/hummingbird/src/components/Navigation/Navigation.js#L177

 Translations need to be added for this copy in all languages and the correct copy passed as the title.